adoptopenjdk-api-v3-updater/src/main/kotlin/net/adoptopenjdk/api/v3/V3Updater.kt

```diff
@@ -130,6 +149,10 @@ class V3Updater {
             }
         } catch (e: Exception) {
             LOGGER.error("Failed to perform full update", e)
+        } catch (e: Throwable) {
```

Comment:
Just wondering what the distinction it between this catch and the one just above. Why do we swallow when it's `Exception` but re-throw when it's `Throwable`? 

Might be worth a comment or consolidating if we actually want to handle them in the same way.

Reply:
Throwable can include unrecoverable errors that we want to log but then allow the VM to fall over and get restarted such as OutOfMemoryError